src/toil_scripts/exome_variant_pipeline/README.md

```diff
@@ -149,3 +159,99 @@ ci-test:
 To run on a distributed AWS cluster, see [CGCloud](https://github.com/BD2KGenomics/cgcloud) for instance provisioning, 
 then run `toil-exome run aws:us-west-2:example-jobstore-bucket --batchSystem=mesos --mesosMaster mesos-master:5050`
 to use the AWS job store and mesos batch system. 
+
+# Methods
+
+## Tools
+
+| Tool     | Version | Description                                                                                       |
+|----------|---------|---------------------------------------------------------------------------------------------------|
+| Samtools | 0.1.19  | Indexes bam files and creates reference index.                                                    |
```

Comment:
Would it be worth linking to the tool containers in quay? I don't necessarily think it would, but I'm just throwing the thought out there in case it inspires anyone.